builders/toolkit/ethereum-api/precompiles/cross-chain-transfers.md

```diff
@@ -0,0 +1,229 @@
+---
+title: Native Cross-Chain Token Transfers
+description: Learn how to use the XCM interface Precompile to transfer tokens from any Tanssi EVM appchain, leveraging their inherent native cross-chain capabilities.
+---
+
+# Native Cross-Chain Token Transfers
+
+## Introduction {: #introduction }
+
+As presented in the [Native Cross-Chain Communication](/learn/framework/xcm/){target=\_blank} article from the Learn section, Tanssi appchains benefit from an inherent capability to communicate and interoperate with any other appchain in the ecosystem. This native cross-chain communication allows safe and fast token transfers leveraging the Cross-Consensus Message format (XCM for short), which facilitates communication between different consensus systems.
+
+The communication protocol enabling token transfers is built on [Substrate](/learn/framework/overview/#substrate-framework){target=\_blank} and runs on a lower level than the EVM, making it harder for EVM developers to access.
+
+This precompile fills the gap between execution layers, exposing a smart contract that abstracts away the underlying complexities, making the execution of cross-chain token transfers as easy as any other smart contract call.
+
+This guide will show you how to interact with the [XCM interface](https://github.com/moondance-labs/tanssi/blob/master/test/contracts/solidity/XcmInterface.sol){target=\_blank} precompile to execute cross-chain token transfers.
+
+The precompile is located at the following address:
+
+```text
+{{networks.dancebox.precompiles.xcmInterface }}
+```
+
+--8<-- 'text/builders/toolkit/ethereum-api/precompiles/security-note.md'
+
+## The XCM Solidity Interface {: #the-xcm-solidity-interface }
+
+The [`XCMInterface.sol`](https://github.com/moondance-labs/tanssi/blob/master/test/contracts/solidity/XcmInterface.sol){target=\_blank} interface on Tanssi EVM appchains is a Solidity interface that allows developers to interact with the precompile's functions.
+
+??? code "XcmInterface.sol"
+
+    ```solidity
+    --8<-- 'code/builders/toolkit/ethereum-api/precompiles/xcm-interface/XcmInterface.sol'
+    ```
+
+The interface includes the necessary data structures along with the following functions:
+
+???+ function "**transferAssetsToPara20**(*uint32* paraId, *address* beneficiary, *AssetAddressInfo[] memory* assets, *uint32* feeAssetItem, *Weight memory* weight) — sends assets to another EVM-compatible appchain using the underlying transfer_assets() transaction included in the module called pallet-xcm"
+
+    === "Parameters"
```

Comment:
I like this a lot! We should do this from now on tbh (cc @eshaben @dawnkelly09 )

Reply:
I think we do something with `transfer_asset()` (cc @dawnkelly09 do you know at the top of your head)?

Reply:
This is the format we've been trying to move towards! Getting there slowly... only the XCM pages have this right now haha

Although @augustocollado added a nice touch with the example tab 💜

Reply:
> I think we do something with `transfer_asset()` (cc @dawnkelly09 do you know at the top of your head)?

I found this in the comments on the contract:
```
/// @dev Function to send assets via XCM to a 20 byte-like parachain 
/// using transfer_assets() pallet-xcm extrinsic.
```
I'm honestly not quite sure what you're referencing by 'do something' with it though.

Reply:
The thing that we do with `transfer_asset()`, because it's a function, is wrap it in back ticks. I think that's what he was trying to hint at